client/header/activity-panel/index.js

```diff
@@ -254,4 +257,19 @@ class ActivityPanel extends Component {
 	}
 }
 
-export default clickOutside( ActivityPanel );
+export default withSelect( select => {
+	const { getCurrentUserData, getNotes, getNotesError, isGetNotesRequesting } = select( 'wc-api' );
+	const userData = getCurrentUserData();
+	const notesQuery = {
+		page: 1,
+		per_page: 3,
```

Comment:
>And do you think we could use per_page: 1 instead of per_page: 3

`per_page` should be 1- accidentally committed `3` from testing 😓 

>What do you think about adding orderby: 'date' just to be sure it doesn't break if we change the default sorting parameter in the future?

The API currently doesn't offer any ordering options for notes.  It is sorted by ID by default.  I had a similar discussion with @allendav previously [here](https://github.com/woocommerce/woocommerce-admin/pull/566#discussion_r226788433).  I'm up for adding in date sorting if you think that's best, but might be easier to handle in another PR if that's okay with you.

Reply:
> might be easier to handle in another PR if that's okay with you

Of course, thanks for explaining that!

Reply:
Issue noted here - https://github.com/woocommerce/woocommerce-admin/issues/1865